dynamic_rest/fields/fields.py

```diff
@@ -118,8 +122,15 @@ def bind(self, *args, **kwargs):
         parent_model = getattr(self.parent.Meta, 'model', None)
 
         remote = is_field_remote(parent_model, self.source)
+
+        def get_model_field():
```

Comment:
Should be fine to use that logic here; it is intended to retrieve remote fields.